readme.txt

```diff
@@ -190,6 +190,15 @@ This version adds an option to refresh the malware scan results on demand, as we
 
 == Changelog ==
 
+= 1.8.21 =
+* Replace goo.gl links
+* Fix post_type pattern match
+* Removed block button from failed logins page
+* Fixed Audit Logs queue timezone issue
+* Added WordPress security recommendations library
+* Added php version check
+* Updated translation file to match the changes
```

Comment:
Should update the changelog to reflect all the changes that are included, but we can do that in a separate PR.